pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.which
+
+
+class OnlyWithCommand(object):
+    def __init__(self, func):
+        self.func = func
```

Comment:
mmm, I've followed the hint of @zachriggle by using the class style decorator.
It is convenient since I can access the command_path attribute within the function in the wrapper code.
f.i.
```
def get_raw_out():
    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
    cmd = [get_raw_out.command_path,"--file",local_path]
    return subprocess.check_output(cmd).decode('utf-8')
```

Using a function it will become something like this:
```
def get_raw_out(cmd_path=''):
    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
    cmd = [cmd_path,"--file",local_path]
    return subprocess.check_output(cmd).decode('utf-8')
```
Basically I need to pass the cmd_path to each function from the decorator. 
i.e.
```
def OnlyWithCommand(function):
    @functools.wraps(function)
    def _OnlyWithCommand(*a, **kw):
        command = function.__module__.split(".")[-1]
        command_path = pwndbg.which.which(command)
        if command_path:
            return function(cmd_path=command_path,*a, **kw)
        else:
            raise OSError('Could not find command %s in $PATH' % command)
    return _OnlyWithCommand


```